### DISCUSSION TITLE
Change '--global-theme-color' in dark mode?

### DISCUSSION THREAD

**Discussion by maxdemirdilek** [alshedivat/al-folio#2528] [Q&A] [answered]
I changed the `--global-theme-color` variable in the `_sass/_themes.scss` file to another color, namely to green. When I view my website and change the theme to dark mode, the theme color seems to be some form of blue, i.e. hyperlinks and the text in the navigation bar are both blue. Is it possible to make it green?

**Comment by george-gca** [comment, marked as answer]
There are 2 `--global-theme-color` variables defined in `_sass/_themes.scss`, one for the light theme and one for the dark theme. Change both and it will be fine.

**Reply by maxdemirdilek** [reply]
Many thanks! Should have found that myself.
